seed_services_client/identity_store.py

```diff
@@ -42,6 +42,19 @@ def get_identity_by_address(self, address_type, address_value):
         params = {"details__addresses__%s" % address_type: address_value}
         return self.session.get('/identities/search/', params=params)
 
+    def get_identity_address(self, identity):
```

Comment:
We could probably have this function declaration be `(self, identity_id, address_type='msisdn')`. That way it's by default MSISDN (which we use the most), but we can still change it if required in the future.

`identity_id` let's us know that we want just the ID for this function, not the whole identity object.